okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -37,7 +34,10 @@
 import io.grpc.internal.KeepAliveManager;
 import io.grpc.internal.SharedResourceHolder;
 import io.grpc.internal.SharedResourceHolder.Resource;
+import io.grpc.okhttp.internal.CipherSuite;
+import io.grpc.okhttp.internal.ConnectionSpec;
```

Comment:
ConnectionSpec is part of grpc's API in this class. We don't want to expose our internal fork, so we let users specify okhttp's ConnectionSpec and then we use Utils.convertSpec() to copy it into an internal equivalent.

Feel free to eagerly convert in `connectionSpec()`; that would allow proguard to remove the references if it isn't specified.